source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -76,6 +74,17 @@
 
 #define QUIC_PREDICT_FALSE_IMPL(x) ABSL_PREDICT_FALSE(x)
 
+#define CHECK(condition)                                                                           \
+  QUIC_LOG_IF_IMPL(FATAL, ABSL_PREDICT_FALSE(!(condition))) << "CHECK failed: " #condition "."
+
+#ifndef NDEBUG
+#define DCHECK(condition) CHECK(condition)
```

Comment:
I'd prefer to avoid this #ifndef.  Can you move the two #define DCHECK to the existing #ifdef NDEBUG above?

Reply:
done